template/src/ontology/run.sh.jinja2

```diff
@@ -11,15 +11,22 @@
 # See README-editors.md for more details.
 
 IMAGE=${IMAGE:-odkfull}
-
+MEMORY_GB=${MEMORY_GB:-8}
```

Comment:
Since this `MEMORY_GB` parameter can be taken from the environment (contrary to `MEMORY_JAVA` which is taken from the ODK configuration file), it should have a name that more explicitly ties it the ODK, e.g. `ODK_MEMORY_GB` (similarly to `ODK_DEBUG`).